docs/_pages/web_client.md

```diff
@@ -220,6 +221,128 @@ web.channels.list((err, res) => {
 
 ---
 
+### Using refresh tokens
+
+If you're using workspace apps, refresh tokens can be used to obtain short-lived access tokens that power your Web API calls from the `WebClient` (this is *required* for distributed apps). To enable the `WebClient` to automatically refresh and swap your access tokens, you need to pass your app's refresh token, client ID, and client secret in the `WebClientOptions`. 
+
+At the time of refresh, the `WebClient` will emit a `token_refreshed` event that will contain the new access token (`access_token`) and time to expiration (`expires_in`). It's recommended to listen to this event and store the access token in a database so you have access to your active token.
+
+```javascript
+const { WebClient } = require('@slack/client');
+
+const refreshToken = process.env.SLACK_REFRESH_TOKEN;
+const id = process.env.SLACK_CLIENT_ID;
+const secret = process.env.SLACK_CLIENT_SECRET;
+
+// Intialize a data structure to store team access token info (typically stored in a database)
+const slackAccessTokens = {};
```

Comment:
i've got multiple suggestions here but i think it would be easier to talk about in person.